### PR TITLE
[ja] sync concepts/architecture/cgroups.md

### PR DESCRIPTION
**Pull request by KKtheGhost**
## PR Summary

- **EN upstream**: `content/en/docs/concepts/architecture/cgroups.md`
- **Sync to**: `content/ja/docs/concepts/architecture/cgroups.md`
- **Updates**:
  - Sync'ed upstream cases for `JDKを利用している場合`

Sync check by`scripts/lsync.sh` on `main` branch:
```diff
$ scripts/lsync.sh content/ja/docs/concepts/architecture/cgroups.md
diff --git a/content/en/docs/concepts/architecture/cgroups.md b/content/en/docs/concepts/architecture/cgroups.md
index 377c073b42..b0a98af660 100644
--- a/content/en/docs/concepts/architecture/cgroups.md
+++ b/content/en/docs/concepts/architecture/cgroups.md
@@ -102,7 +102,12 @@ updated to newer versions that support cgroup v2. For example:
  Update these agents to versions that support cgroup v2.
 * If you run [cAdvisor](https://github.com/google/cadvisor) as a stand-alone
  DaemonSet for monitoring pods and containers, update it to v0.43.0 or later.
-* If you use JDK, prefer to use JDK 11.0.16 and later or JDK 15 and later, which [fully support cgroup v2](https://bugs.openjdk.org/browse/JDK-8230305).
+* If you deploy Java applications, prefer to use versions which fully support cgroup v2:
+    * [OpenJDK / HotSpot](https://bugs.openjdk.org/browse/JDK-8230305): jdk8u372, 11.0.16, 15 and later
+    * [IBM Semeru Runtimes](https://www.eclipse.org/openj9/docs/version0.33/#control-groups-v2-support): jdk8u345-b01, 11.0.16.0, 17.0.4.0, 18.0.2.0 and later
+    * [IBM Java](https://www.ibm.com/docs/en/sdk-java-technology/8?topic=new-service-refresh-7#whatsnew_sr7__fp15): 8.0.7.15 and later
+* If you are using the [uber-go/automaxprocs](https://github.com/uber-go/automaxprocs) package, make sure
+  the version you use is v1.5.1 or higher.

 ## Identify the cgroup version on Linux Nodes  {#check-cgroup-version}

content/ja/docs/concepts/architecture/cloud-controller.md is still in sync
```
Sync check by`scripts/lsync.sh` on `update/architecture-cgroups` branch:
```diff
$ scripts/lsync.sh content/ja/docs/concepts/architecture/cgroups.md
content/ja/docs/concepts/architecture/cgroups.md is still in sync
```

Thanks to the reviewers in advance.
Best Regards,

Kivinsae Fang

### CODE REVIEW

**Review by t-inu — COMMENTED** [review]
Thank you. I left one comment.

File: content/ja/docs/concepts/architecture/cgroups.md [hunk 1]
```diff
@@ -84,7 +84,12 @@ cgroup v2はcgroup v1とは違うAPIを利用しているため、cgroupファ
 * サードパーティーの監視またはセキュリティエージェントはcgroupファイルシステムに依存していることがあります。
  エージェントをcgroup v2をサポートしているバージョンに更新してください。
 * Podやコンテナを監視するために[cAdvisor](https://github.com/google/cadvisor)をスタンドアローンのDaemonSetとして起動している場合、v0.43.0以上に更新してください。
-* JDKを利用している場合、[cgroup v2を完全にサポートしている](https://bugs.openjdk.org/browse/JDK-8230305)JDK 11.0.16以降、またはJDK15以降を利用することが望ましいです。
+* Javaアプリケーションをデプロイする場合は、完全にcgroup v2をサポートしているバージョンを利用してください：
+    * [OpenJDK / HotSpot](https://bugs.openjdk.org/browse/JDK-8230305): jdk8u372、11.0.16、15以降
+    * [IBM Semeru Runtimes](https://www.eclipse.org/openj9/docs/version0.33/#control-groups-v2-support): jdk8u345-b01、11.0.16.0、17.0.4.0、18.0.2.0以降
+    * [IBM Java](https://www.ibm.com/docs/en/sdk-java-technology/8?topic=new-service-refresh-7#whatsnew_sr7__fp15): 8.0.7.15以降
+* [uber-go/automaxprocs](https://github.com/uber-go/automaxprocs)パッケージを利用している場合は、
+利用するバージョンがv1.5.1以上であることを確認してください。
```
[the remark this answers is not in the document]
Comment (atoato88) [reply content/ja/docs/concepts/architecture/cgroups.md:92]: Agree with this comment.

**Comment by atoato88** [comment]
@KKtheGhost 
Could you update this with the comment above?

**Comment by KKtheGhost** [comment]
> @KKtheGhost Could you update this with the comment above?

Hi @atoato88 
The comment is updated and squashed into the PR commit.
Thanks so much for the review.

Best Regards.

**Comment by atoato88** [comment]
/lgtm
Thank you for the quick update!

**Comment by atoato88** [comment]
/assign @nasa9084 
Could you review this?

**Review by nasa9084 — COMMENTED** [review]
/approve

**Pull request merged**